pages/api/FolderApi.tsx

```diff
@@ -1,17 +1,17 @@
-import getDate from "./GetData";
+import requestApi from "./RequestApi";
 
-export async function getFolderUserData() {
-  return getDate("/users/1");
+export async function getUserData(DEFAULT_USER_ID = 1) {
+  return requestApi(`/users/${DEFAULT_USER_ID}`);
```

Comment:
오.. 근데 DEFAULT_USER_ID가 상수가 아니라 그냥 파라미터 이름으로 들어갔군요.
DEFAULT 라면, 아래처럼 작성하는 게, fallback으로서의 의미에 맞는 사용일거에요

```suggestion
const DEFAULT_USER_ID  = '1' ;
export async function getUserData(userId = DEFAULT_USER_ID) {
  return requestApi(`/users/${userId}`);
```